### DISCUSSION TITLE
Skywalking Multi-tenant

### DISCUSSION THREAD

**Discussion by chenhaipeng** [apache/skywalking#7474] [Q&A]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- Why does skywalking not provide agent config custom attributes, I need custom attributes to isolate tenants, for example, agent.tags=userId, backend uses userId to isolate data

___

**Comment by wu-sheng** [comment]
In an open source community, this means we don't have contributors want to do this, and no committer says we would like to maintain if the contributor(s) are gone.
Nothing special, all existing features are following the same logic.
